packages/amplify_auth_plugin_interface/lib/src/SignIn/SignInWithWebUIRequest.dart

```diff
@@ -18,12 +18,12 @@ import 'package:flutter/foundation.dart';
 import './AuthProvider.dart';
 
 class SignInWithWebUIRequest {
-  AuthProvider provider;
+  AuthProvider? provider;
   SignInWithWebUIRequest({this.provider});
   Map<String, dynamic> serializeAsMap() {
     final Map<String, dynamic> pendingRequest = <String, dynamic>{};
     if (this.provider != null) {
-      pendingRequest["authProvider"] = describeEnum(provider);
+      pendingRequest["authProvider"] = describeEnum(provider!);
     }
```

Comment:
what is the right usage here `this.provider` or `provider`. We should stick to the one type of usage.

Reply:
Either way is fine.  "this" is added to make it more explicit that we are using an instance variable vs. some local or method parameter variable.  

We have no set standard for this throughout all of our plugins.  If we want to stick to one type of usage we should probably come together as a team and decide on a common path forward before making the change to all of our plugins.